conv_rnn/test.py

```diff
@@ -17,19 +18,19 @@ def main():
     parser.add_argument("--gpu_number", default=0, type=int)
     args = parser.parse_args()
 
-    model.set_seed(5, no_cuda=args.no_cuda)
-    data_loader = data.SSTDataLoader(args.data_dir)
+    model.set_seed(3, no_cuda=args.no_cuda)

```

Comment:
make seed configurable?

Reply:
Actually the seed is useless in test.py, I'll remove it later.